src/supporting-data/taxonomy/components/results/__tests__/TaxonomyCard.spec.tsx

```diff
@@ -0,0 +1,13 @@
+import renderWithRouter from '../../../../../shared/__test-helpers__/customRender';
+import TaxonomyCard from '../TaxonomyCard';
+
+import mockData from '../../../__mocks__/taxonomyModelData';
+
+describe('Taxonomy Card tests', () => {
```

Comment:
This should contain a `'should allow card selection and navigation'` test as for all the other card renderers for other namespaces, this would have caught this issue.